addons/smtp-mail.sh

```diff
@@ -104,14 +104,14 @@ esac
 # Enter your SMTP username
 if yesno_box_yes "Does $MAIL_SERVER require any credenitals, like username and password?"
 then
-    MAIL_USERNAME=$(input_box_flow "Please enter the SMTP username to your email provider.\nE.g. you@$MAIL_SERVER")
+    MAIL_USERNAME=$(input_box_flow "Please enter the SMTP username to your email provider.\nE.g. you@mail.com")
 
     # Enter your mailuser password
     MAIL_PASSWORD=$(input_box_flow "Please enter the SMTP password to your email provider.")
 fi
 
 # Enter the recipient
-RECIPIENT=$(input_box_flow "Please enter the recipient email address that shall receive all mails.\nE.g. recipient@$MAIL_SERVER")
+RECIPIENT=$(input_box_flow "Please enter the recipient email address that shall receive all mails.\nE.g. recipient@mail.com")
```

Comment:
Why not use the variable?

Reply:
Because
https://github.com/nextcloud/vm/pull/1509#discussion_r499128287

Reply:
Bit that has nothing to do with the example. 

IMHO it's nicer to show the server. 

But ok. :+1: